ReClass.NET/Memory/ProcessInfo.cs

```diff
@@ -26,6 +26,11 @@ public ProcessInfo(IntPtr id, string name, string path)
 			{
 				using (var i = NativeMethods.GetIconForFile(Path))
 				{
+					if (i == null)
```

Comment:
Could you change this to `return i?.ToBitmap();`?

Reply:
im not hugely familiar with c#, does this check if i is null before calling ToBitmap?

Reply:
yeah its incredibly usefull, https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operators